pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1476,7 +1476,7 @@ func checkIfDiskReadyToUseFunc(filename string) (bool, error) {
 		return false, err
 	}
 	if (info.Mode() & os.ModeDevice) != 0 {
-		file, err := os.OpenFile(filename, os.O_RDONLY, 0777)
+		file, err := os.OpenFile(filename, os.O_RDONLY, 0600)
```

Comment:
Same as above.

Reply:
> Same as above.

Here, for Read-Only mode, the Permission setting is not being checked. So changed it to 600 to keep gosec from complaining.